tools/integration_tests/rename_dir_limit/rename_dir_limit_test.go

```diff
@@ -37,12 +39,22 @@ func TestMain(m *testing.M) {
 
 	flags := [][]string{{"--rename-dir-limit=3", "--implicit-dirs"}, {"--rename-dir-limit=3"}}
 
+	setup.ExitWithFailureIfBothTestBucketAndMountedDirectoryFlagsAreSet()
+
 	if setup.TestBucket() != "" && setup.MountedDirectory() != "" {
```

Comment:
We can also move this check under that method, and please rename it accordingly.

Reply:
We can't because for readonly tests condition changed.

Reply:
Do we need this check here, since we exit already if both flag are set?

Reply:
Sorry function was for when both the flags not set. updated the name.